chapters/hpmor-chapter-046.tex

```diff
@@ -221,6 +221,7 @@
 
 “No,” she finally said. “Please, Harry, don’t ask any more.”
 
+% sic
```

Comment:
Please can you put the `% sic` directly after the odd word (almost as if it were in the text, in the conventional way: "this seems wrnog [sic]"?